CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ All notable changes to this project are documented in this file.
 
  - Make a visible error message when the `slint-lsp` panics
 
+### Added
+
+ - `ProgressIndicator` widget.
+
```

Comment:
Please move this into the "Slint Language" section as "- Added `ProgressIndicator` widget." instead of a separate section.